app/handlers/analysis.py

```diff
@@ -18,4 +26,76 @@ class AnalysisHandler(tornado.web.RequestHandler):
     @apiError error_message The error message to display.
     """
     def post(self):
+        identifier = self.request.body_arguments["identifier"]
+
+        self.objectTrack(identifier)
+        self.safetyAnalysis(identifier)
+
+        message = "Hello,\n\tWe have finished processing your video and identifying any dangerous interactions.\nThank you for your patience,\nThe Santos Team"
+        subject = "Your video has finished processing."
+
+        EmailHelper.send_email(self.request.body_arguments["email"], subject, message)
+
         self.finish("Analysis")
+
+    def safetyAnalysis(self, identifier, prediction_method=None):
```

Comment:
Maybe don't duplicate these functions and call the functions from the other handlers?